atlasdb-config/src/main/java/com/palantir/atlasdb/factory/AtlasDbDialogueServiceProvider.java

```diff
@@ -185,9 +191,12 @@ private <T> T createDialogueProxy(Class<T> type, Channel channel) {
         return AtlasDbHttpClients.createDialogueProxy(taggedMetricRegistry, type, channel);
     }
 
-    private static <T> T wrapInProxy(Class<T> type, T service) {
-        return RetryOnSocketTimeoutExceptionProxy.newProxyInstance(
-                type, () -> FastFailoverProxy.newProxyInstance(type, () -> service));
+    private <T> T wrapInProxy(Class<T> type, T service) {
+        return UnknownRemoteDebuggingProxy.newProxyInstance(
+                type,
+                timeLockServerListConfig.map(ServerListConfig::servers),
```

Comment:
Is there a reason we want to have a separate `Refreshable<T>` passed in into each of the proxying clients for a given ADDSP, as opposed to just one shared across them? (Possible this is actually fine if there's some clever magic in the refreshables library.)

Reply:
I assume you're referencing the fact that I'm re-mapping it each time? Yeah, I can unify that into one.

Reply:
exactly, yeah that was what I was referring to